WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Create Shipping Label Form/Customs/ShippingLabelCustomsFormInputViewModel.swift

```diff
@@ -55,7 +59,13 @@ final class ShippingLabelCustomsFormInputViewModel: ObservableObject {
     ///
     private let currency: String
 
-    init(customsForm: ShippingLabelCustomsForm, countries: [Country], currency: String) {
+    /// Whether ITN validation is required.
+    ///
+    private lazy var itnValidationRequired: Bool = {
```

Comment:
nit: I'm a little confused by the comment and name here. I see this is used for the `missingITNForDestination` validation below. Does this property represent when validation is required, or when the ITN is required? And is it specific to when the ITN is required for the destination country, or will it also be used for validation when the ITN is required because the item value is greater than $2500?

The way it's being used now, it feels like it's specifically for whether the ITN is required for the destination, and could be named `itnRequiredForDestination`. (But the current naming might make more sense to me once the validation implementation is complete!)

Reply:
Thank you! The suggested naming is so much clearer, I've updated it in 52c62da64fa51f0b5645c59f99d84d915f694234.